README.md

```diff
@@ -342,16 +345,6 @@ www.youtube.com/watch?v=zbjub8BQPyE)
 * [Kafka Streams](https://kafka.apache.org/documentation/streams/) ![](https://img.shields.io/github/stars/apache/kafka.svg?style=social) - Kafka client library for buliding applications and microservices where the input and output are stored in kafka clusters
 * [Spark Streaming](https://spark.apache.org/streaming/) ![](https://img.shields.io/github/stars/apache/spark.svg?style=social) - Micro-batch processing for streams using the apache spark framework as a backend supporting stateful exactly-once semantics
 
-## Outlier and Anomaly Detection
```

Comment:
These sections are being removed potentially as you still need to update from master